component/admin/controllers/icals.php

```diff
@@ -369,7 +372,7 @@ function save($key = null, $urlVar = null){
 			$icsFileid = $icsFile->store();
 			$message = JText::_( 'ICS_FILE_IMPORTED' );
 		}
-		if (JRequest::getCmd("task") != "icals.reloadall")
+		if (JRequest::getCmd("task") !== "icals.reloadall" && $guest !== 1)
```

Comment:
since there is no view for icals.reloadall does this not throw a Joomla error?  Surely what we should be doing is suppressing the $message as opposed to blocking the reload